lib/vanagon/component.rb

```diff
@@ -290,6 +290,10 @@ def get_build_dir
       end
     end
 
+    def get_dependency_hash
+      { name => { 'version' => version, 'ref' => options[:ref] }.delete_if { |k, v| !v } }
```

Comment:
What is this magic?

Why not just build a simple hash like we had it?

We need 2 keys in a hash, that's it.

I don't even know what this does by looking at it.

Whatever :man_shrugging: